src/sonic_ax_impl/mibs/ietf/rfc2737.py

```diff
@@ -287,6 +287,7 @@ def reinit_data(self):
         self.physical_name_to_oid_map = {}
         self.pending_resolve_parent_name_map = {}
 
+        Namespace.connect_all_dbs(self.statedb, mibs.STATE_DB)
```

Comment:
>connect_all_dbs [](http://example.com/codeflow?start=18&length=15)

`SonicV2Connector_Native::connect` is heavy, can we minimize the operation, for example, only re-connect when exception caught.

I am not sure if `DBInterface::_onetime_connect` properly handle resource, there may be resource leak. We need to add test.

Reply:
Fixed, re-connect when exception happen.
I will create new UT in swss-common for DBInterface::_onetime_connect resource issue.